### ISSUE TITLE
[calendar] Delete categories  

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#449]
```
Roundcube - V 0.8.4

/**
 * Jquery Calendar
 * @version 9.9.6.9.8.1 - 19.11.2012
 ...
 * Plugin to add a calendar to RoundCube.
 * @version 0.2 BETA 2
 * @author Lazlo Westerhof
 ...
 **/

Using CalDAV provider and 
SabraDAV ( Server: V 1.7.4 / RC PlugIn: V 2.6 - 04-11-2012) backend:

When I delete a category, all entries are deleted!
Result is an empty calender after syncing with SabraDAV server.



```

Original issue reported on code.google.com by `alois.kr...@wdns.at` on 7 Dec 2012 at 2:58

**Comment by GoogleCodeExporter** [comment]
```
I'm not sure what you are doing. If you delete a category connected to a CalDAV 
all events will be wiped out from database table 'events_caldav' but they still 
are present in your CalDAV backend. 'events_caldav' is nothing else than a 
cache.

NOTE: If you are using sabredav plugin: This plugin is designed for SabreDAV 
1.6.x. It has not been ported yet to SabreDAV 1.7.x !!!
```

Original comment by `roland.l...@gmail.com` on 7 Dec 2012 at 3:18

**Comment by GoogleCodeExporter** [comment]
```
No feedback. Closing soon.
```

Original comment by `roland.l...@gmail.com` on 10 Dec 2012 at 4:20

**Comment by GoogleCodeExporter** [comment]
```
>> NOTE: If you are using sabredav plugin: This plugin 
>> is designed for SabreDAV 1.6.x. It has not been 
>> ported yet to SabreDAV 1.7.x !!!

http://myroundcube.com/myroundcube-plugins/sabredav-plugin :
"...Head over to Sabredav download section in their project page and grab the 
latest version available..."

SabreDAV 1.7.4 is in use!
I know there is a different table style in SabreDAV 1.7.x. 
I´ll check this.


```

Original comment by `alois.kr...@wdns.at` on 10 Dec 2012 at 4:34

**Comment by GoogleCodeExporter** [comment]
```
We missed to update our documentation page when SabreDAV 1.7.x was released. 
Sorry for that. We will place a note there asap. I let the ticket open until 
the plugin has been ported to SabreDAV 1.7.x (on Top of our ToDo's).
```

Original comment by `roland.l...@gmail.com` on 12 Dec 2012 at 5:53
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in sabredav 3.0. Don't miss to read our update notes:

http://myroundcube.com/myroundcube-plugins/sabredav-plugin



```

Original comment by `roland.l...@gmail.com` on 15 Dec 2012 at 1:42
- Changed state: **Fixed**
